src/pages/Signup/Validation.js

```diff
@@ -0,0 +1,26 @@
+export function chkEmail(email) {
+  var regExp =
+    /^[0-9a-zA-Z]([-_.]?[0-9a-zA-Z])*@[0-9a-zA-Z]([-_.]?[0-9a-zA-Z])*.[a-zA-Z]{2,3}$/i;
+  return regExp.test(email) ? true : false;
```

Comment:
이미 test라는 함수자체가 boolean을 리턴합니다 굳이 삼항연산자 쓸 필요 없습니다!
지속적으로 같은 부분 보이는 것 같은데 boolean의 특성에 대해서 다시 한번 공부해보시면 좋을 것 같습니다!
```suggestion
  return regExp.test(email)
```

Reply:
수정 + 확인 완료!! 좀 더 공부해보겠습니다!!😭